<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cron√≥metro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pbSalir">
      <property name="text">
       <string>&amp;Salir</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Q</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLCDNumber" name="lcdTime">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="smallDecimalPoint">
           <bool>true</bool>
          </property>
          <property name="digitCount">
           <number>6</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Puerto Serial</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QComboBox" name="cbSerialPort"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Controles</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QPushButton" name="pbHabilitar">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>&amp;Habilitar</string>
               </property>
               <property name="shortcut">
                <string>Ctrl+H</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pbReiniciar">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>&amp;Reiniciar</string>
               </property>
               <property name="shortcut">
                <string>Ctrl+R</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="pbDetener">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>&amp;Detener</string>
               </property>
               <property name="shortcut">
                <string>Ctrl+D</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTableWidget" name="twTiempo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="rowCount">
        <number>15</number>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Vuelta</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tiempo Vuelta</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tiempo Total</string>
        </property>
       </column>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbSalir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
